AUTHORS

```diff
@@ -5,3 +5,4 @@ Miguel Beltran <m@beltran.work>
 Neevash Ramdial <mail@neevash.dev>
 Andrew Teich <andrewteich@me.com>
 Volodymyr Buberenko <v.buberenko@gmail.com>
+Om Phatak <everythingoutdated@gmail.com>
```

Comment:
Please do not edit the AUTHORS file.

We actually should simply remove it since there is no consistency on who is added and who is not.

Reply:
AUTHORS file are mostly inconsistent anyways even in the official Flutter and Firebase repos.

Wouldn't removing them would also mean to remove the license comments at top of files referring credits to members in this file?

Reply:
Please check on that once and let me know what to do.

Reply:
AUTHORS file has nothing to do with licensing. This repository is also not owned by Google, but by the Flutter Community, so what they do is not relevant for us.

> Wouldn't removing them would also mean to remove the license comments at top of files referring credits to members in this file?

This definitely needs a cleanup IMO, but it is a different topic.

Do as you wish, I don't want to waste time discussing this.